macros/dmt_unit_test.sql

```diff
@@ -1,42 +1,44 @@
-{%- test unit_test(model, input_mapping, expected_output, name, description, compare_columns, depends_on) -%}
+{%- test unit_test(model, input_mapping, expected_output, name, description, compare_columns, exclude_columns, depends_on) -%}
     {%- set test_sql = dbt_datamocktool.get_unit_test_sql(model, input_mapping, depends_on)|trim -%}
-    {%- set test_report = dbt_datamocktool.test_equality(expected_output, name, compare_model=test_sql, compare_columns=compare_columns) -%}
+    {%- set test_report = dbt_datamocktool.test_equality(expected_output, name, compare_model=test_sql, compare_columns=compare_columns, exclude_columns=exclude_columns) -%}
     {{ test_report }}
 {%- endtest -%}
 
-{% test unit_test_incremental(model, input_mapping, expected_output, name, description, compare_columns, depends_on) %}
+{% test unit_test_incremental(model, input_mapping, expected_output, name, description, compare_columns, exclude_columns, depends_on) %}
     {%- set test_sql = dbt_datamocktool.get_unit_test_incremental_sql(model, input_mapping, depends_on)|trim -%}
-    {%- set test_report = dbt_datamocktool.test_equality(expected_output, name, compare_model=test_sql, compare_columns=compare_columns) -%}
+    {%- set test_report = dbt_datamocktool.test_equality(expected_output, name, compare_model=test_sql, compare_columns=compare_columns, exclude_columns=exclude_columns) -%}
     {{ test_report }}
 {% endtest %}
 
-{%- macro test_equality(model, name, compare_model, compare_columns=None) -%}
+{%- macro test_equality(model, name, compare_model, compare_columns=[], exclude_columns=[]) -%}
 
-    {#-- Prevent querying of db in parsing mode. This works because this macro does not create any new refs. #}
+    -- Prevent querying of db in parsing mode. This works because this macro does not create any new refs.
     {%- if not execute -%}
         {{ return('') }}
     {%- endif -%}
 
-    {#- setup -#}
+    -- Setup
```

Comment:
Forgive me for changing the formatting here, my editor was struggling

| Before | After |
| --- | --- |
| <img width="852" alt="Screenshot 2023-03-29 at 08 54 08" src="https://user-images.githubusercontent.com/32493490/228465765-95e20dfd-440c-492b-bea5-01f2aac57151.png"> | <img width="858" alt="Screenshot 2023-03-29 at 08 54 18" src="https://user-images.githubusercontent.com/32493490/228465817-9b3cfa9b-b59d-4f88-9250-2e6416f188be.png"> |